### DISCUSSION TITLE
Link from my gallery doesn't show images

### DISCUSSION THREAD

**Discussion by Scorpions95** [chevereto/chevereto#20] [Q&A]
Only shows "images"
![Screenshot 2023-03-23 19 39 59](https://user-images.githubusercontent.com/24534657/227327202-1131a387-7c36-4c4a-8b9c-857f4f2732ce.png)
![Screenshot 2023-03-23 19 40 59](https://user-images.githubusercontent.com/24534657/227327228-35970da0-bbbb-42f3-9328-9abd5fc3eea5.png)
![Screenshot 2023-03-23 19 41 47](https://user-images.githubusercontent.com/24534657/227327270-604b03b5-672c-4a87-ba06-d4dfc745f637.png)
on free different forums
how can i fix it?
@rodber

**Comment by rodber** [comment]
Chevereto doesn't serve these static files, your server does. The server we provide doesn't limit any traffic, you need to check your systems.
